src/ucp/core/ucp_ep.h

```diff
@@ -388,7 +388,7 @@ typedef struct ucp_conn_request {
     ucp_listener_h              listener;
     uct_conn_request_h          uct_req;
     uct_iface_h                 uct_iface;
-    ucp_wireup_sockaddr_data_t  client_data;
+    ucp_wireup_sockaddr_data_t  sockaddr_data;
```

Comment:
IMHO 'data' is enough for the param name

Reply:
IMO `data` is too common

Reply:
Also true. We don't usually call the type and its param with the same name. but maybe in this case it's ok..

Reply:
in new pieces of code I call it `sa_data`, wdyt?

Reply:
yes, sa_data sounds better.